openedx/features/partners/views.py

```diff
@@ -38,3 +38,22 @@ def register_user(request, slug):
         return views.Give2AsiaRegistrationView.as_view()(request, partner=partner)
     except ImportError:
         raise Http404('Your partner is not properly registered')
+
+
+@require_http_methods(["POST"])
+@sensitive_post_parameters('password')
+def login_user(request, slug):
+    """
+    This is general login view, for users of all partners
+    :param request.
+    :param slug: partner slug
+    :return: JsonResponse object with success/error message
+    """
+    partner = get_object_or_404(Partner, slug=slug)
+
+    try:
+        views = import_module(PARTNERS_VIEW_FRMT.format(slug=partner.slug))
```

Comment:
How about writing a single method to import module dynamically and raise `raise Http404('Your partner is not properly registered')` if got ImportError. and reuse it in all the views..